src/manager.cpp

```diff
@@ -567,21 +569,10 @@ int boot(const po::variables_map& vm) {
         ignored_modules = vm["ignore"].as<std::vector<std::string>>();
     }
 
-    // NOLINTNEXTLINE(concurrency-mt-unsafe): not problematic that this function is not threadsafe here
-    const char* mqtt_server_address = std::getenv("MQTT_SERVER_ADDRESS");
-    if (mqtt_server_address == nullptr) {
-        mqtt_server_address = "localhost";
-    }
-
-    // NOLINTNEXTLINE(concurrency-mt-unsafe): not problematic that this function is not threadsafe here
-    const char* mqtt_server_port = std::getenv("MQTT_SERVER_PORT");
-    if (mqtt_server_port == nullptr) {
-        mqtt_server_port = "1883";
-    }
-
-    MQTTAbstraction& mqtt_abstraction = MQTTAbstraction::get_instance(mqtt_server_address, mqtt_server_port);
```

Comment:
I'm not sure, if we should leave the possibility to use environment variables for configuring the mqtt server, otherwise one would need to touch the config every time. But on the other hand, if we will need it again, we can add it back

Reply:
I've re-added this to the runtime settings which should reduce the code duplication whilst retaining this feature